src/main/scala/com/dwolla/cloudflare/DnsRecordClient.scala

```diff
@@ -93,7 +91,7 @@ case class MultipleCloudflareRecordsExistForDomainNameException(domainName: Stri
      | - ${records.mkString("\n - ")}
      |
      |This resource refuses to process multiple records because the intention is not clear.
-     |Clean up the records manually or enhance the custom resource Lambda to handle multiple records.""".stripMargin)
+     |Clean up the records manually or update your code to handle multiple records.""".stripMargin)
```

Comment:
This is nitpicky but it's not the code using this library that would need to be updated: it's the library itself. This exception is thrown on line 45 if an existing DNS record is looked up by name (e.g. www.dwolla.com) and multiple records are returned from Cloudflare. The library doesn't know how to handle that situation, so it throws the exception. This error message is trying to be explicit about the implementation choice that was made, i.e., to give up instead of trying to figure out which of the multiple records should be updated.